### DISCUSSION TITLE
Cyfrin Updraft server down?

### DISCUSSION THREAD

**Discussion by LordAdhaar** [Cyfrin/foundry-full-course-cu#2927] [General]
Are the cyfrin updraft servers experiencing downtime for everyone?

**Comment by Vinay12345-neutron** [comment]
Yeah same issue

**Comment by EngrPips** [comment]
The Engineering team are aware and actively working on it.

**Reply by LordAdhaar** [reply]
Maybe the course is doing too well and there are too many students lol

**Reply by EngrPips** [reply]
Yeah, Maybe.

**Comment by abhinavreddyyy** [comment]
Even I'm facing the same issue but you can continue the course on their youTube  channel

**Reply by armaansidana2003** [reply]
Yup, im doing the same

**Reply by LordAdhaar** [reply]
Same
